### PR TITLE
SEO-264 update map repo find candidates function

### PR DESCRIPTION
**Pull request by micromax14**
https://timeoutgroup.atlassian.net/browse/SEO-264

### CODE REVIEW

**Comment by pharman** [comment]
MAX MILAZZO!!

**Comment by enekomunoz** [comment]
Hello Max!! Hi Paul!
Who else is around?
Why am I still getting these emails?

**Comment by llewellynthomas** [comment]
@enekomunoz @pharman  we should probably remove you from the emails :).

Hope you are both doing well. 

@enekomunoz how is your little one doing?

@flatnine & @benjambles  are the only other developers left that you will know.

**Comment by micromax14** [comment]
Hi both, I hope you are doing fine!

File: Entity/MapRepository.php [hunk 1]
```diff
@@ -127,4 +137,62 @@ public function findOneForUrlOrPath($url, $path)
...
+    }
+
+    /**
+     * Build map from array.
+     *
+     * @param array $data
+     * @return Map
+     * @throws \Exception
+     */
+    public function buildMapFromArray(array $data)
```
Comment (llewellynthomas) [inline Entity/MapRepository.php]: @micromax14  This should not be in a Repository class?

Comment (llewellynthomas) [reply Entity/MapRepository.php]: This should be an toArray method on the entity class

Comment (micromax14) [reply Entity/MapRepository.php]: yes, I forgot to move that. thanks

**Review by llewellynthomas — APPROVED** [review]

**Pull request merged**